web/assets/scss/bundle_card.scss

```
@@ -0,0 +1,236 @@
+.bundle-card-line {
+    width: 100%;
+    height: 225px;
+    background: #fff;
+    margin: 0 0 40px 0;
+    border: solid 1px #ccc;
+    border-radius: 5px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+    padding: 25px 20px;
+    overflow: hidden;
+
+    &:hover {
+        box-shadow: 0 7px 15px #ddd;
+    }
+}
+
+.bundle-card-line-data {
+    height: 185px;
+    float: left;
+    overflow: hidden;
+}
+
+.bundle-card-line-information {
+    float: left;
+    overflow: hidden;
+}
+
+.bundle-card-line-information h2 {
+    font-size: 19px;
+    color: #333;
+    font-weight: 600;
+    margin-top: 20px;
+    word-break: break-word;
+}
+
+.bundle-card-line-description {
+    font-size: 15px;
+    color: #63605d;
+    font-weight: 400;
+}
+
+.bundle-card-line-description p {
+    padding: 0;
+    margin: 0;
+}
+
+.bundle-card-line-counters {
+    width: 25%;
+    float: left;
+    overflow: hidden;
+}
+
+.bundle-card-line-counters ul {
+    padding: 10px 35px;
+    margin: 0;
+    list-style-type: none;
+}
+
+.bundle-card-line-counters ul li {
+    display: block;
+    height: 40px;
+    line-height: 40px;
+    border-bottom: 1px solid #d3d3d3;
+    padding: 0 5px 0 0;
+}
+
+.bundle-card-line-counters ul li:last-of-type {
+    border-bottom: none;
+}
+
+.bundle-card-line-counters ul li i {
+    color: #2a9dc7;
+    width: 25px;
+    text-align: center;
+    font-size: 18px;
+}
+
+.bundle-card-line-authors {
+    width: 100%;
+    overflow: hidden;
+}
+
+.bundle-card-line-owner-label {
+    display: block;
+    font-size: 15px;
+}
+
+.bundle-card-line-owner-author {
+    font-size: 16px;
+    font-weight: 600;
+}
+
+.bundle-card-line-owner-name {
+    position: relative;
+    top: -5px;
+    width: 75%;
+    float: left;
+    margin-left: 10px;
+    color: #63605d;
+}
+
+.bundle-card-line-owner-photo {
+    width: 34px;
+    height: 34px;
+    float: left;
+}
+
+.bundle-card-line-owner-photo img {
+    width: 100%;
+}
+
+.bundle-card-line-maintainer {
+    margin: 0 20px 0 10px;
+    float: left;
+}
+
+.bundle-card-line-maintainer-username {
+    float: left;
+    padding-left: 5px;
+}
+
+.bundle-card-line-href {
+    width: 70px;
+    height: 100%;
+    float: left;
+    overflow: hidden;
+}
+
+.bundle-card-line-href a {
+    display: block;
+    width: 100%;
+    height: 100%;
+    background: #cbcbcb;
+    color: #f4f4f4;
+    line-height: 225px;
+    text-align: center;
+    font-size: 40px;
+    font-weight: 400;
+}
+
+.bundle-card-line-href a:hover {
+    background: #f15a22;
```

This is our primary color
```suggestion
    background: $primary;
```